src/dswell/subcmds/create.py

```diff
@@ -14,55 +14,18 @@ def touch_file(filepath):
 
 
 @click.command()
```

Comment:
Please at least manually verify what AI did to the codebase, see what it did?

The creation command is gone; it is making the directory for deletion 

```
python3 -m dswell list

Pending Deletions:
--------------------------------------------------------------------------------
Path                                               Created At           Time Left 
--------------------------------------------------------------------------------
/Users/proshan/dswell                              13:08:35             59m50s    
--------------------------------------------------------------------------------
```